packages/shared/src/components/post/SharePostContent.tsx

```diff
@@ -36,7 +36,7 @@ function SharePostContent({
           href={
             post.sharedPost.source.id === 'unknown'
               ? post.sharedPost.permalink
-              : post.sharedPost.commentsPermalink
+              : `${post.sharedPost.commentsPermalink}?squad=${post.source.id}&n=${post.source.name}`
```

Comment:
Can't we just use squad handle and also mask the query params with `as`? Masking also makes sure nobody will copy the link so no danger of broken links if squad handle changes. Don't know if we want users to actually copy this link anyway, it is more of a internal app thing. WDYT?

Reply:
Only internal for now, how do you mean masking with as?
Not sure i follow correctly

Reply:
Using `as` prop from nextjs `Link`, something like this:
```jsx
<Link
  href={`${post.sharedPost.commentsPermalink}?squad=${post.source.id}&n=${post.source.name}`}
  as={post.sharedPost.commentsPermalink}
>
  <a>...</a>
</Link>
```

Reply:
Oh wasnt aware of this! Feel free to adjust in the PR if you want 👍
Is it a web thing or a specific react thing?

Reply:
Hmm looked at it now, we use `_blank` for opening those links, is that intentional? Because since `as` is nextjs router thing (the browser just sees the final URL) so the masking will not work with `_blank`.

Reply:
Sorry for not seeing it while reviewing initially.

Reply:
I think we can make it conditional of its internal or not, could you please check with @idoshamun?
Else I'll check tomorrow 🙏

Reply:
Because that is the mechanism to show "Back to squad" link when you open a shared post that is from daily.dev. 

<img width="738" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/5a69cbc2-a27a-4bdf-92dc-ad6ef48ecbce">

Reply:
Oh don't mind me, I thought it's a share externally component. I suggest we follow Ante's advice. It doesn't makes sense anymore to open it in new tab because we have this nice utility button now

Reply:
@idoshamun / @capJavert I think we should only redirect without blank to internal links right?

- Links to `post.sharedPost. commentsPermalink` then don't add _blank
- Is external only `post.sharedPost.permalink` then do add _blank